### ISSUE TITLE
Capacitor goole map infowindow

### ISSUE THREAD

**Issue by parvinsamin** [ionic-team/capacitor-google-maps#54]
## Bug Report

 
 
Plugin Capacitor/google-maps
infowindow bug
v = 6.0.0
 
Hello, I want to report a bug in the Google Maps section. Unfortunately, there isn't sufficient documentation for using InfoWindow in this library. I wasn't able to resolve this with my own knowledge. Could you please look into it and provide a practical example? I think this might be a bug.

**Comment by dKolovosSmartnav** [comment]
Hello @parvinsamin!

It seems that the infoWindow is not supported. I am trying to find an alternative. However If you wish to display simple text you can use 'snippet' while creating the marker.

**Comment by j2d2** [comment]
I hit this same issue using Capacitor v7 (@capacitor/google-maps@7.0.0) - there is no InfoWindow showing up when clicking a Marker. This means that we are unable to show any UI near the Marker on the Map - a very serious issue for this package.

I tried using an Ionic Popover, but without the $event we can't connect the popover to the HTML element location on the page.

Additionally, the setOnMarkerClickListener returns a title but the `snippet` field appears to always return a blank string, so that might be an additional bug - but without a way to see the InfoWindow, the snippet is of little value here.

**Comment by kolobosdimitrios** [comment]
@j2d2 I agree with you. After several days I just couldn't find a way around this!

**Comment by mengoni** [comment]
my ugly workaround for the snippets:

`private snippets: { [key: string]: string } = {};
`

and when I add the markers to the map, I also add the snippets to the hashmap using coordinates as key:


```
const key = `${latitude},${longitude}`;
this.snippets[key] = 'my snippet text';
```

and, to retrieve the snippets:

```
await this.map.setOnMarkerClickListener(async (marker) => {
        const key = `${marker.latitude},${marker.longitude}`;
        console.log('snippet text: ' + this.snippets[key]);
      });
```
